autocertkit/utils.py

```diff
@@ -1726,33 +1724,52 @@ def deploy_two_droid_vms(session, network_refs, sms=None):
     make_vm_noninteractive(session, vm1_ref)
     make_vm_noninteractive(session, vm2_ref)
 
-    log.debug("Setup vms on network")
-    i = 0
+    return (host_master_ref, host_slave_ref, vm1_ref, vm2_ref)
+
+
+def config_network_for_droid_vm(session, vm_ref, network_ref, did, sms=None):
+    """Setup VM network"""
+
+    device = 'eth%d' % did
+
+    log.debug("Setting interfaces up for %s" % device)
+    # Note: only remove all existing networks on first run.
+    setup_vm_on_network(session, vm_ref, network_ref, device, wipe=(did == 0))
```

Comment:
What does `did` stand for?

Reply:
It's not my design, it's always there. 
did is device id, e.g. 0 of eth0, 1 of eth1

eth0 is special, Framework will remove (wipe) eth0 and re-create it